packages/clerk-js/src/ui/components/OrganizationSwitcher/OrganizationSwitcherPopover.tsx

```diff
@@ -144,8 +149,9 @@ export const OrganizationSwitcherPopover = React.forwardRef<HTMLDivElement, Orga
                   organization={currentOrg}
                   user={user}
                   sx={theme => t => ({ padding: `0 ${theme.space.$6}`, marginBottom: t.space.$2 })}
+                  aria-hidden='true'
                 />
-                <Actions>
+                <Actions role='group'>
```

Comment:
Why did you add this here?

Reply:
Our code treats these actions and the `OrganizationActionList` as 2 different groups

i believe it is allowed to have groups inside a `menu`